src/appmixer/hibob/employees/DeletedEmployee/component.json

```
@@ -0,0 +1,25 @@
+{
+    "name": "maesn.hibob.employees.DeletedEmployee",
```

Appmixer vendor is expected when the connector is in `src/appmixer` folder.
```suggestion
    "name": "appmixer.hibob.employees.DeletedEmployee",
```